File: 2nd Largest/main.js
function secondLargest(arr) {
    arr = [...new Set(arr)]
    return sort(arr)[arr.length-2];
}

function sort(arr){
    for(let i=0;i<arr.length;i++){
        let min = arr[i];

        for(let j=i;j<arr.length;j++){
            if(arr[j] < min){
                let temp = arr[i];
                arr[i] = arr[j];
                arr[j] = temp;
            }
        }
    }
    // console.log(arr);
    return arr;
}

console.log(
    secondLargest([20, 139, 94, 67, 31]) 
)

File: 2nd Largest/main.py
def second_largest(arr):
    arr = list(set(arr))
    return sort(arr)[len(arr) - 2]

def sort(arr):
    for i in range(len(arr)):
        min_index = i

        for j in range(i + 1, len(arr)):
            if arr[j] < arr[min_index]:
                min_index = j

        temp = arr[i]
        arr[i] = arr[min_index]
        arr[min_index] = temp

    return arr

print(
    second_largest([20, 139, 94, 67, 31])
)